docs/content/upgrade-guides/admin/1-0-0.md

```diff
@@ -0,0 +1,40 @@
+# Medusa Admin: 1.0.0
+
+Medusa Admin has been moved from a standalone repository to our mono-repository to centralise projects and ship it as an NPM package.
+
+## Overview
+
+The NPM package name is `@medusajs/admin-ui` and in our core repository the UI can be found within `packages/admin-ui`. 
+
+This guide will cover three cases where the move might affect your current setup:
+
+- Updating your existing project to use the plugin
+- Replacing your already deployed admin with the plugin
+- Preserving customizations in the plugin
+
+## Actions required
+
+**Updating your existing project to use the plugin**
+
+As mentioned, Medusa Admin is now distributed via NPM and installed as a plugins. To replace your existing admin, you first need to install the plugin in your Medusa server project.
+
+Follow our quickstart guide [INSERT LINK] to get started setting up the plugin.
+
+**Replacing your already deployed admin with the plugin**
+
+The move to an NPM package has implications for the workflow to deploy the admin. You can still use a platform like Vercel or Netlify to host the project, but that requires you to eject the admin system from the plugin instead of serving it on the server.
+
```

Comment:
I think the wording might be a bit confusing here. There is no need to eject the admin when hosting on Vercel etc.